README.md

```diff
@@ -1,3 +1,4 @@
 "# kodilla" 
 "# kodilla" 
 "# kodilla" 
+"# kodilla" 
```

Comment:
co to jest?

Reply:
Miałem problem z utworzeniem PR i zrobiłem commit pliku readme.md